.github/workflows/ttrt-flatbuffer-ttnn.yml

```diff
@@ -0,0 +1,106 @@
+        ]
+    name: ${{ matrix.build.name }}
+    runs-on: ${{ matrix.build.runs-on }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+          fetch-depth: 0 # Fetch all history and tags
+
+    - name: Install build dependencies
+      uses: ./.github/actions/install-deps
+      with:
+        os: ${{ matrix.build.runs-on }}
+
+    - name: Build and cache ttmlir-toolchain
+      uses: ./.github/actions/build-toolchain
+      with:
+        os: ${{ matrix.build.runs-on }}
+        sdk: ${{ env.SDK_VERSION }}
+
+    - name: Set reusable strings
+      id: strings
+      shell: bash
+      run: |
+        echo "build-output-dir=${{ github.workspace }}/build" >> "$GITHUB_OUTPUT"
+
+    - name: Configure CMake
+      shell: bash
+      run: |
+        source env/activate
+        cmake -G Ninja \
+        -B ${{ steps.strings.outputs.build-output-dir }} \
+        -DCMAKE_CXX_COMPILER=${{ matrix.build.cpp_compiler }} \
+        -DCMAKE_C_COMPILER=${{ matrix.build.c_compiler }} \
+        -DCMAKE_BUILD_TYPE=${{ matrix.build.build_type }} \
+        -DTTMLIR_ENABLE_RUNTIME=${{ matrix.build.enable_runtime }} \
+        -DTTMLIR_ENABLE_RUNTIME_TESTS=${{ matrix.build.enable_runtime }} \
+        -DTT_RUNTIME_ENABLE_PERF_TRACE=${{ matrix.build.enable_perf }} \
+        -S ${{ github.workspace }}
+
+    - name: Set perf env variables
+      if: ${{ matrix.build.enable_perf }} == 'ON'
+      shell: bash
+      run: |
+        export ENABLE_TRACY=1
+
+    - name: Build
+      shell: bash
+      run: |
+        source env/activate
+        cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build.build_type }}
+
+    - name: Run Test
+      shell: bash
+      run: |
+        source env/activate
+        export SYSTEM_DESC_PATH=${{ github.workspace }}/test/system_desc/${{ matrix.build.runs-on }}.ttsys
+        llvm-lit -v ${{ github.workspace }}/build/test/ttmlir/Silicon
+        cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build.build_type }} -- check-ttmlir
+
+    - name: Build ttrt
+      shell: bash
+      run: |
+        source env/activate
+        cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build.build_type }} -- ttrt
+
+    - name: Run ttrt functional tests
+      if: ${{ matrix.build.enable_perf }} == 'OFF'
+      shell: bash
+      run: |
+        source env/activate
+        pip install loguru
+        pip install torch
+        pip install pandas
+        pip install seaborn
+        ttrt run ${{ steps.strings.outputs.build-output-dir }}/test/ttmlir/Silicon
+
+    - name: Run ttrt performance tests
+      if: ${{ matrix.build.enable_perf }} == 'ON'
+      shell: bash
+      run: |
+        source env/activate
+        pip install loguru
+        pip install torch
+        pip install pandas
+        pip install seaborn
+        ttrt perf ${{ steps.strings.outputs.build-output-dir }}/test/ttmlir/Silicon
```

Comment:
It's good to have this stubbed out, but I think we can hold off on getting perf tests through and save the cycles for now.  I just don't want to end up with pipelines that no one is looking at, once we care about perf regressions we can enable this job.

Reply:
sounds good